src/js/index.js

```diff
@@ -77,13 +83,24 @@ function handleSort() {
 		.forEach(sortedChild => contentEl.append(sortedChild));
 }
 
+function handleExpired() {
+	if (showExpired.checked) {
+		search.set('expired', 'y');
```

Comment:
I think we should use booleans (i.e. `expired=true|false`) here, and handle it as part of `updateUrl`

Reply:
@aslafy-z I think using booleans for a boolean param would be better - what was your reasoning for using `y` or `n`?

Reply:
Agree with you on that @vikaspotluri123 💯

Reply:
go for `expired=true`. See https://github.com/swapagarwal/swag-for-dev/pull/159#issuecomment-451221219 https://github.com/swapagarwal/swag-for-dev/pull/159#issuecomment-451371846, I proposed to ditch any value and @itaisteinherz proposed to use `y` or `n` instead.

Reply:
@aslafy-z Sorry for blaming you 😅 I'll update this in a bit.

Reply:
Done in 83ac6376f9fbc20b8756828a3eb5e702d6c27b29 👌🏻